src/components/CardContent/CardContent.js

```diff
@@ -37,17 +37,36 @@ const CardContent = ({
   const cardLinkContentArray = Object.keys(cardLinkContent);
   const cardInfoContentArray = Object.keys(cardInfoContent);
 
+  if (
+    isValidElement(cardIcon) &&
+    iconDescription !== CardContent.defaultProps.iconDescription
+  ) {
+    // eslint-disable-next-line no-console
+    console.warn(
```

Comment:
Could use `invariant` or `warning` here, I think, just so we can get the dev-time checks and removal in prod

Reply:
Great point @joshblack! - Did it: https://github.com/carbon-design-system/carbon-components-react/pull/528/commits/fc24f4fbae8527cf32b5c76e997e8667a7017051